backend/src/main/java/se/chalmers/dat265/group1/storage/repository/GenericRepository.java

```diff
@@ -181,9 +185,9 @@ public void deleteObject(int id) {
             try {
                 result = type.newInstance();
             } catch (InstantiationException e) {
-                e.printStackTrace();
+                log.error("getMatchJSON",e);
             } catch (IllegalAccessException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 187, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)